### DISCUSSION TITLE
v0.106.1 currency field 0 value display issue [Jun 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5887] [Discord Threads]
> 🙋‍♂️ **CJ-JackSys**  
## Why when I put 1.00 under purchase price does it then disappear after screen refresh?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1104135845407101050/image.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### created a github issue   
#### https://github.com/nocodb/nocodb/issues/5607  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1104135845591662704)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
